ansible/roles/postgres-migration/files/sunbird_programs/V4.1.0.sql

```diff
@@ -2,3 +2,4 @@ INSERT INTO "public"."configuration" ("key", "value", "status") VALUES ('program
 ALTER TABLE program ADD COLUMN targetCollectionPrimaryCategories jsonb;
 CREATE TYPE programTargetType AS ENUM ('collections', 'searchCriteria');
 ALTER TABLE program ADD COLUMN target_type programTargetType;
+UPDATE "public"."configuration" SET value='VidyaDaan: Your Content $contentName for the project $projectName has been approved by the project owner with few changes. Please login to $url for details.' WHERE key='smsContentAcceptWithChanges';
```

Comment:
This will throw migration error right? Different hash of the file

Reply:
No this wont as the migration for this V4.1.0.sql  file is not done on any of the vdn instances.

Reply:
Okay